docs/extend/extend-apiml/api-mediation-security.md

```diff
@@ -189,12 +185,21 @@ The ticket request requires the token in one of the following formats:
 - Cookie named apimlAuthenticationToken.
 - Bearer authentication
 
-The request takes one parameter named `applicationName`, the name of the application for which the PassTicket should be generated. This parameter must be supplied.
+The request takes the `applicationName` parameter, which is the name of the application for which the PassTicket should be generated. This parameter must be supplied.
 
 The response is a JSON object, which contains information associated with the ticket.
 
 For more details, see the OpenAPI documentation of the API Mediation Layer in the API Catalog.
 
+#### Supported authentication methods
+The API Mediation Layer provides multiple methods which clients can use to authenticate. When the API ML is run as a part of Zowe, all of the following methods are enabled and supported. All the methods are supported at least to some extent with each authentication provider.
+
+##### Username/Password
+The client can authenticate via Username and password. There are multiple methods which can be used to deliver the credentials. There are more details in the [ZAAS Client documentation](https://docs.zowe.org/stable/extend/extend-apiml/api-mediation-security.html#zaas-client).
+
+##### JWT Token
+When the client authenticates with the API ML, it receives in exchange the JWT token which can be used for further authentication. If z/OSMF is configured as the authentication provider and the client already received a JWT token produced by the z/OSMF, it is possible to reuse this token within the API ML for authentication.
```

Comment:
May be we can say it this way: "When the client application authenticates with the API ML, it receives in exchange a JWT token which can be used for further access to mainframe services and resources. This token can be then used to effectively utilize the SSO (Single Sign On) capability of the API ML."

A-Note-Not-for-the-document: It doesn't matter if the token was produced by z/OSMF. Any token returned by API ML ZAAS is good for SSO.